Feature/port screenshot

wholewheattoast opened the pull request:
Port lettuce-webdriver screenshoting to planterbox.

Since we don't have the notion of a terrain file, gathering some details from config, and getting date test is run inside function.

npilon commented on planterbox_webdriver/screenshot.py:
```diff
@@ -45,13 +45,26 @@ def capture_screenshot_delay(test, delay):
+    import pytz
+    
+    now = datetime.datetime.now(pytz.utc)
+    test_run_date = now.strftime("%Y-%m-%d")
```
This would mean all the screenshots from every test on the same day get dumped into a single directory. That seems less than ideal?

wholewheattoast commented:
@npilon now getting time (for base) from planterbox config rather then setting independently.

npilon commented on planterbox_webdriver/screenshot.py:
```diff
@@ -45,13 +45,20 @@ def capture_screenshot_delay(test, delay):
+    base = '{}/{}'.format(
+        test.config['screenshot.dir'][0],
+        test.config['start_time'][0],
+    )
```
`os.path.join` can take multiple path segments to join together. Prefer that for joining path segments to `.format()`.

wholewheattoast commented:
@npilon https://github.com/npilon/planterbox-webdriver/commit/46e3143b9195fb2cba074e06cce5ff09c66a07fd addresses feedback

https://github.com/npilon/planterbox-webdriver/commit/9a3c44dcd7da6646a78bd18114082393b89928ed does same for pull request against planterbox.
